schemas/2019-09-07/Microsoft.Kusto.json

```diff
@@ -0,0 +1,621 @@
+﻿{
```

Comment:
Please see https://github.com/Azure/azure-resource-manager-schemas#submitting-a-pr

You're missing adding a link to all the root schemas - without this, customers won't be able to use this resource in a template.

Reply:
Thanks. Should be resolved.